test/Listener/OptionsListenerTest.php

```
@@ -324,7 +324,7 @@ public static function invalidMethodsMethodOnlyProvider(): array
      * @param string $method
      */
     #[DataProvider('invalidMethodsWithoutAllowProvider')]
-    public function testListenerReturnsNullIfNotAnHttpRequest($method, array $matchParams)
+    public function testListenerReturnsNullIfNotAnHttpRequest($method, array $matchParams): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$method'. (undefined)

(UnusedFormalParameter)